pyro/contrib/autoguide/__init__.py

```diff
@@ -417,6 +418,8 @@ def quantiles(self, quantiles, *args, **kwargs):
         :rtype: dict
         """
         loc, scale = self._loc_scale(*args, **kwargs)
+        if loc is None or scale is None:
+            raise ValueError('loc and scale cannot be None')
```

Comment:
Thanks for being safe. How about:
```py
if scale is None:
    raise NotImplementedError('scale cannot be computed analytically')
```

Reply:
this is in the `AutoContinuous` class so should error if either `loc` or `scale` isnt computed with a general error message. do you want an overridden check in `AutoTransformedNormal`?

Reply:
I just think we should raise a `NotImplementedError` rather than a `ValueError`. Feel free to `raise NotImplementedError` with an empty error message.